pkg/apis/ibmcloud/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -214,7 +214,7 @@ func (in *IbmcloudMachineProviderSpec) DeepCopyInto(out *IbmcloudMachineProvider
 	*out = *in
 	out.TypeMeta = in.TypeMeta
 	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
-	out.Spec = in.Spec
+	in.Spec.DeepCopyInto(&out.Spec)
```

Comment:
Does the update of the file auto-generated?

Besides, I have add some new files to check gofmt, golint at `hack/xxx`, we can run those scripts manually for now, but I will put them to makefile later.